invenio_search/ext.py

```diff
@@ -311,24 +325,32 @@ def create_index(
                 final_alias = build_alias_name(index, prefix=prefix, app=self.app)
             index_result = (
                 final_index,
-                self.client.indices.create(
-                    index=final_index,
-                    body=json.load(body),
-                    ignore=ignore,
-                )
-                if not dry_run
-                else None,
+                (
```

Comment:
This change is to keep black happy. And it is still complaining :(

Reply:
do you use locally the same version as github does?

Reply:
Yep, same version of black, pytest-black, and running on  py thon3.9
```
% pip freeze |grep black
black==24.1.1
pytest-black==0.3.12
```
It is interesting that it works now in 3.7 and fails in 3.8 and 3.9. The thing is that it keeps asking for more '('. If I add one, it wants two. I add two, and then it asks for three.

Reply:
if it works in python3.7 and not in python>3.8 then you are allowed to remove python3.7 from the test matrix. because black >= 24.0.0 can't be installed on python3.7 and therefore with black==23.X.Y works the old styling. 

i think there is a bug in black. it handles this situation really strange. i tried the file on my machine and it added also a second (), but if i removed the second () then it was also ok.

maybe you can solve the problem by moving the if else block out would maybe also increase readability?

Reply:
Thanks a lot for looking into this, @utnapischtim. I have the feeling that we did run into an issue with black. 

I like your suggestion of playing a bit with the code. Since both parts of the `if.. else` had the first item in the tuple, I took that one out, and put it only in the return value. Now, black is happy in all the versions :-)

Reply:
only minor comment. you should also change the default values of `index_result` and `alias_result` https://github.com/inveniosoftware/invenio-search/blob/e933b1af89850792eaaedf720f34b3649eada8d6/invenio_search/ext.py#L314-L315

happy to help :)

Reply:
THanks! I've removed part of the initialization, and cleaned up the other one.